python2/test_typing.py

```diff
@@ -26,10 +26,7 @@
 from typing import Pattern, Match
 import typing
 import weakref
-try:
-    import collections.abc as collections_abc
-except ImportError:
-    import collections as collections_abc  # Fallback for PY3.2.
+import collections
```

Comment:
Isn't this backwards? In newer Python versions you *always* need to use `collections.abc.Iterable` (etc.). You get `DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working`

Reply:
This is _Python 2_

Reply:
Also it is already the correct way in the Python 3 version.

Reply:
D'oh.